lib/web_ui/test/text/canvas_paragraph_builder_test.dart

```diff
@@ -8,7 +8,7 @@ import 'package:test/test.dart';
 import 'package:ui/src/engine.dart';
 import 'package:ui/ui.dart';
 
-const String paragraphStyle = 'style="position: absolute; white-space: pre-wrap; overflow-wrap: break-word; overflow: hidden;"';
+const String paragraphStyle = 'font-family: sans-serif; position: absolute; white-space: pre-wrap; overflow-wrap: break-word; overflow: hidden;';
```

Comment:
do we still need overflow hidden?

Reply:
It's only needed for certain cases like when `max-lines` is set. I'll change it so that the `overflow` property is only added when needed, instead of all the time.